functions/createReview/createSuggestions/regexRules.js

```diff
@@ -0,0 +1,37 @@
+export default [
+  {
+    id: 'PROMISE_ALL_USAGE',
+    regex: /\bPromise\.all\b/i,
+    description:
+      'Executing Promise.all(items.map(async => { … })) leads to the creation of an undefined number of Promises, each executing something asynchronous and possibly saturating the event loop and consuming much memory. Use a library like sindresorhus/p-map or sindresorhus/p-all instead.'
```

Comment:
Looks like there's a `too` missing between `consuming` and `much memory`.

Reply:
Thank you, fixed.